.bazelrc

```diff
@@ -4,6 +4,9 @@
 
 startup --host_jvm_args=-DBAZEL_TRACK_SOURCE_DIRECTORIES=1
 
+# Needed for rules_js
+common --enable_runfiles
```

Comment:
this is no longer needed in current rules-js, are we testing with an old one?

Reply:
I got an error without it. minimum dep is 1.34.0 here and latest 1.x is 1.42.0... we'll be on rules_js 2.0 here soon so can drop this flag at that point